README.Md

```diff
@@ -15,10 +15,16 @@ Spring while the frontend will be implemented using the React framework
 * Maven
 * Docker
 ## Running the app
-* `mvn spring-boot:build-image`
-* `docker run --env-file YOUR_ENVIRONMENT_FILE -p 8080:8080 mfsl-backend:0.0.1-SNAPSHOT`
 
-`YOUR_ENVIRONMENT_FILE` should be replaced with the absolute path to a file with real environment variables.
+### Docker
```

Comment:
Wrong order. You need to build image before running it. Also you need to source env file to do the mvn build-image